push_manifest.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+set -e
```

Comment:
nit: Please place this file under `contrib/`, probably `contrib/scripts/` is the best spot.

Reply:
Done